README.md

```diff
@@ -2,83 +2,48 @@
 
 ## Assignment
 
-- Build an awesome weather app that shows the weather at your current  location.
+- Build an awesome movieapp that shows popular and high rated movies.
 - Code it for the platform (Android, iOS, web) you applied for or the one you prefer.
 
-## Minimum requirements
-- Show the weather for your current location
-- Use data from provided API (see API DOCS)
-- Good usability
-- Be visually appealing and follow platform conventions
-- Budget: 8-12 hours
+## Requirements
+
+- Use an open https://developers.themoviedb.org/open source API
+- Discover most popular and highly rated movies
+- Display the movies with creative look and feel of an app to meet design guidelines for your platform (Material etc)
+- Launch a detail screen whenever a particular movie is selected
+- Allow user to save a favorite movie for offline access
+- Allow user to read movie reviews
 
 ## Assets
+
 You can find some icon assets at [https://github.com/erikflowers/weather-icons](https://github.com/erikflowers/weather-icons) that you can use if you please. You are more than welcome to use other assets that you feel are better fitted for your solution. We do not expect you to be a designer, but since application development on Daresay most often includes visual interfaces we do expect you to to be well aware of your platforms design guidelines and conventions.
```

Comment:
These icons are not of use for movies :)

Reply:
Yeah removed that section :D